api/src/test/java/com/basistech/rosette/api/RosetteAPITest.java

```diff
@@ -233,22 +233,22 @@ void testMatchRecord(String testFilename, String responseStr, int statusCode) th
         }
     }
 
-    private static Stream<Arguments> testMatchRecordMissingFieldParameters() throws IOException {
-        return getTestFiles("-record-similarity-missing-field.json");
-    }
-
-    @ParameterizedTest(name = "testFilename: {0}; statusCode: {2}")
-    @MethodSource("testMatchRecordMissingFieldParameters")
-    void testMatchRecordMissingField(String testFilename, String responseStr, int statusCode) throws IOException {
-        setStatusCodeResponse(responseStr, statusCode);
-
-        try {
-            readValueRecordMatcher(testFilename);
-            fail("Did not throw exception for a field type in request but not in mapping");
-        } catch (IllegalArgumentException e) {
-            assertEquals("Unsupported field name: primaryName not found in field mapping", e.getMessage());
-        }
-    }
+//    private static Stream<Arguments> testMatchRecordMissingFieldParameters() throws IOException {
```

Comment:
We can still check whether we get a "primaryName not found in field mapping" response, so this test code shouldn't be commented out. But instead of catching an exception, since it doesn't throw an exception anymore, we can check `responseStr` content like this:

```
        readValueRecordMatcher(testFilename);
        assertEquals("{\n"
                + "  \"code\": \"badRequestFormat\",\n"
                + "  \"message\": \"Unsupported field name: primaryName not found in field mapping\",\n"
                + "  \"stack\": null\n"
                + "}", responseStr);
```

Reply:
test has been updated